components/cdc/src/initializer.rs

```diff
@@ -266,7 +266,14 @@ impl<E: KvEngine> Initializer<E> {
             DownstreamState::Initializing | DownstreamState::Stopped
         ));
 
+        let mut scan_long_time = false;
         while !done {
+            // Add metrics to observe long time incremental scan region count
```

Comment:
The function can return in the `while` loop. So please consider to use `defer` to decrease the metric.